templates/guides/migrating-to-v0.8.0.md.tmpl

```diff
@@ -0,0 +1,166 @@
+---
+page_title: "Migrating to v0.8.0"
+subcategory: "Guides"
```

Comment:
I know this is a guide, but should we group these under it's own `Migration` subcategory?

Reply:
How about "Migrations"?